lib/src/row/Row.tsx

```diff
@@ -19,9 +18,9 @@ export default function Row({ gutter = "none", align, justify, wrap = true, reve
 }
 const StyledRow = styled.div`
   display: flex;
-  flex-direction: ${({ reverse }: RowProps) => (reverse ? "row-reverse" : "row")};
-  flex-wrap: ${({ wrap }: RowProps) => (wrap ? "wrap" : "nowrap")};
-  align-items: ${({ align }: RowProps) => {
+  flex-direction: ${({ reverse }: RowPropsType) => (reverse ? "row-reverse" : "row")};
+  flex-wrap: ${({ wrap }: RowPropsType) => (wrap ? "wrap" : "nowrap")};
+  align-items: ${({ align }: RowPropsType) => {
```

Comment:
It is not necessary to indicate `RowPropsType` in all cases if you indicate it to styled components.

Reply:
Done!